config.yaml

```diff
@@ -1,4 +1,4 @@
 options:
   snap-channel:
-    default: "latest/stable"
+    default: "reef/stable"
```

Comment:
so this goes back to the discussion about using same tracks for charm-microceph and microceph. I feel we should stick to using latest/stable in main. And on the release branches use relevant channels.

Reply:
It was set in the card that it should be a named branch rather than latest.